src/client/app/actions/monaco.ts

```diff
@@ -27,7 +27,7 @@ export class MonacoActionTypes {
 export class ChangeTabAction implements Action {
     readonly type = MonacoActionTypes.CHANGE_TAB;
 
-    constructor(public payload: string, public language: string) { }
+    constructor(public payload: { name: string, language: string }) {}
```

Comment:
we should be sending multiple parameters when possible. Is there a reason to this?